config/environments/development.rb

```diff
@@ -15,7 +15,7 @@
 
   # Don't care if the mailer can't send
   config.action_mailer.raise_delivery_errors = true
-  config.action_mailer.delivery_method = :test
+  config.action_mailer.delivery_method = :letter_opener
   config.action_mailer.perform_deliveries = false
```

Comment:
I was playing around with it. It seemed to only work for me if you changed `perform_deliveries = true`. Did you get it working?